tests/common/util.rs

```diff
@@ -751,6 +751,15 @@ impl AtPath {
         }
     }
 
+    pub fn read_symlink(&self, path: &str) -> String {
```

Comment:
Is this still used? The new tests don't seem to include it (unless I'm missing something).

Reply:
Yes, here:
https://github.com/niyaznigmatullin/coreutils/blob/9166269aec682adf2b42a4f0abe5c76296f6e3ec/tests/by-util/test_cp.rs#L2078

And here:
https://github.com/niyaznigmatullin/coreutils/blob/9166269aec682adf2b42a4f0abe5c76296f6e3ec/tests/by-util/test_cp.rs#L2044

Reply:
Thanks! GH wasn't hightlighting them. I should've used ctrl+f :)